test/utils/version/version.go

```diff
@@ -0,0 +1,29 @@
+package version
```

Comment:
missing copyright boilerplate (`hack/boilerplate.go.txt`), please change the copyrright year